var/spack/repos/builtin/packages/py-sphinx-inline-tabs/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxInlineTabs(PythonPackage):
+    """Add inline tabbed content to your Sphinx documentation."""
+
+    homepage = "https://github.com/pradyunsg/sphinx-inline-tabs"
+    pypi = "sphinx_inline_tabs/sphinx_inline_tabs-2022.1.2b11.tar.gz"
+
+    maintainers = ["JBlaschke"]
+
+    version(
+        "2022.1.2b11", sha256="afb9142772ec05ccb07f05d8181b518188fc55631b26ee803c694e812b3fdd73"
+    )
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-flit-core", type="build")
+
```

Comment:
Missing deps on:
```
requires-python = ">=3.8"
requires = ["sphinx >= 3"]
```

Reply:
See my comments above. I'm starting to think that you want me to do something else, but since I'm new to build Spack packages, it's not clear to me what you're recommending.

Reply:
Might want to read through https://spack.readthedocs.io/en/latest/packaging_guide.html

Reply:
I did. Not only is it very long, it focuses on explaining how things work rather than giving practical advise on how to translate a makefile/setup.py/etc... So it's more of a developer guide rather than a manual. I guess what I would like to see is a exhaustive list of example.

Anyway I think I understand this now, and will make the changes.

(By the way, I found this more helpful: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html)